gitman/models/config.py

```diff
@@ -1,3 +1,4 @@
+# pylint: disable=attribute-defined-outside-init
```

Comment:
* The problem :
I wanted to avoid set to None internal attributes first to None and then just after to the right value

* The solution :
I will re-enable  attribute-defined-outside-init and do something like this

```
self._path = None
self.path = self.config_path

```